docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,129 @@
+# Quickstart
+
+<p class="description">Get started building a Toolpad app.</p>
+
+This page will guide you into creating your very first Toolpad application. You will create a "hello world" type of application that will ask for a name and print a greeting.
+
+## Prerequisites
+
+Make sure you have a running Toolpad instance.
```

Comment:
When I run the commands as shown here, I get this error:

`ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker/compose/docker-compose.yml'`

What am I doing wrong? Is there something I'm missing here? (I'm clueless when it comes to Docker but I do have it installed and running.)

Reply:
https://github.com/mui/mui-toolpad/pull/859#discussion_r968851183